<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="addEntryGroupBox">
      <property name="title">
       <string>Add link</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="addEntryGrid">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="urlInput">
           <property name="placeholderText">
            <string>Enter URL from YouTube</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QPushButton" name="addButton">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>62</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Download&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>resources/ic_add_white_48px.svg</normaloff>resources/ic_add_white_48px.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="downloadLocationInput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Download video to ...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="fileDialogInput">
           <property name="minimumSize">
            <size>
             <width>88</width>
             <height>31</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select download location...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="downloadItemGrid">
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="videoThumb">
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightblue;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="videoTitle">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="videoFormatSelector"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="videoAuthor">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="videoViews">
           <property name="text">
            <string>Views</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="videoDuration">
           <property name="text">
            <string>Duration</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="videoDownloadSize">
           <property name="text">
            <string>Download size</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color:  rgb(35, 94, 255)</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>512</width>
         <height>258</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="downloadListLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionVisit_us"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionVisit_us">
   <property name="text">
    <string>Visit us</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
